package.json

```diff
@@ -10,17 +10,22 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "seed": "mongoimport --db bandland --collection users --file userData.json --jsonArray",
+    "gen-data": "node --max-old-space-size=8192 database/mockDataGen.js",
     "start": "nodemon server/index.js",
     "test": "jest --verbose",
     "test:coverage": "jest --verbose --coverage"
   },
   "dependencies": {
     "body-parser": "^1.18.3",
     "cors": "^2.8.5",
+    "csv-write-stream": "^2.0.0",
     "express": "^4.16.4",
+    "faker": "^4.1.0",
     "grunt": "^1.0.4",
     "jquery": "^3.3.1",
+    "knex": "^0.16.5",
```

Comment:
were `knex` and` pg` being used somewhere in this codebase? I notice that they're being added as packages, but I don't think I saw them being used in this commit

Reply:
I am using pg now but not knex. I went ahead and uninstalled knex. Thanks for catching that.